### PR TITLE
flutter: improve session and screenshots docs

### PR DESCRIPTION
**Pull request by buenaflor**
## DESCRIBE YOUR PR
Update snippets and wording

## IS YOUR CHANGE URGENT?  

Help us prioritize incoming PRs by letting us know when the change needs to go live.
- [ ] Urgent deadline (GA date, etc.): 
- [ ] Other deadline: 
- [x] None: Not urgent, can wait up to 1 week+

## SLA

- Teamwork makes the dream work, so please add a reviewer to your PRs.
- Please give the docs team up to 1 week to review your PR unless you've added an urgent due date to it.
Thanks in advance for your help!

## PRE-MERGE CHECKLIST

*Make sure you've checked the following before merging your changes:*

- [ ] Checked Vercel preview for correctness, including links
- [ ] PR was reviewed and approved by any necessary SMEs (subject matter experts)
- [ ] PR was reviewed and approved by a member of the [Sentry docs team](https://github.com/orgs/getsentry/teams/docs)

## LEGAL BOILERPLATE

Look, I get it. The entity doing business as "Sentry" was incorporated in the State of Delaware in 2015 as Functional Software, Inc. and is gonna need some rights from me in order to utilize my contributions in this here PR. So here's the deal: I retain all rights, title and interest in and to my contributions, and by keeping this boilerplate intact I confirm that Sentry can use, modify, copy, and redistribute my contributions, under Sentry's choice of terms.

## EXTRA RESOURCES

- [Sentry Docs contributor guide](https://docs.sentry.io/contributing/)

### CODE REVIEW

File: platform-includes/configuration/auto-session-tracking/flutter.mdx [hunk 1]
```diff
@@ -1,29 +1,23 @@
...
-  await SentryFlutter.init(
-    (options) => options.autoSessionTrackingInterval = const Duration(milliseconds: 60000)
-    appRunner: () => runApp(MyApp()),
-  );
-}
+```dart
+SentryFlutter.init((options) {
+  // Change timeout duration until session is terminated in background
+  options.autoSessionTrackingInterval = const Duration(seconds: 60)
+}, appRunner: () => runApp(MyApp()));
```
Comment (buenaflor) [inline platform-includes/configuration/auto-session-tracking/flutter.mdx:11]: I am unsure whether we should leave the appRunner in there, it is an optional parameter

File: docs/platforms/flutter/configuration/options.mdx [hunk 2]
```diff
@@ -151,6 +151,7 @@ This option can be overridden using <PlatformIdentifier name="in-app-include" />
 <ConfigKey name="attach-screenshot">
 
 Takes a screenshot of the application when an error happens and includes it as an attachment.
+Enable this feature by setting <PlatformIdentifier name="attach-screenshot" /> to `true` and wrapping your root widget with `SentryWidget(child: MyApp())`.
```
Comment (coolguyzone) [inline docs/platforms/flutter/configuration/options.mdx:154]:
```suggestion
Enable this option by setting <PlatformIdentifier name="attach-screenshot" /> to `true` and wrapping your root widget with `SentryWidget(child: MyApp())`.
```

File: docs/platforms/flutter/configuration/releases.mdx [hunk 3]
```diff
@@ -62,14 +62,14 @@ After configuring your SDK, you can install a repository integration or manually
 
 Monitor the [health of releases](/product/releases/health/) by observing user adoption, usage of the application, percentage of [crashes](/product/releases/health/#crash), and [session data](/product/releases/health/#session). Release health will provide insight into the impact of crashes and bugs as it relates to user experience, and reveal trends with each new issue through the [Release Details](/product/releases/release-details/) graphs and filters.
 
+In order to monitor release health, the SDK sends session data.
+
 <Note>
 
-Crash reporting and app hang detection are not available for watchOS.
+Release health in Flutter is only available for Android, iOS and macOS.
```
Comment (coolguyzone) [inline docs/platforms/flutter/configuration/releases.mdx:69]:
```suggestion
Release health in Flutter is only available for Android, iOS, and macOS.
```

File: platform-includes/configuration/auto-session-tracking/flutter.mdx [hunk 4]
```diff
@@ -1,29 +1,23 @@
 To benefit from the health data, you must use at least version 4.0.0 of the Flutter SDK.
 
-By default, the session is terminated once the application is in the background for more than 30 seconds. You can change the time out with the option named `sessionTrackingIntervalMillis`. It takes the amount in milliseconds. For example, to configure it to be 60 seconds:
+#### Session Timeout
 
-```dart
-import 'package:flutter/widgets.dart';
-import 'package:sentry_flutter/sentry_flutter.dart';
+By default, the session is terminated once the application is in the background for more than `30 seconds`. You can change the default timeout with the option `autoSessionTrackingInterval` to a `Duration` of your choosing.
```
Comment (coolguyzone) [inline platform-includes/configuration/auto-session-tracking/flutter.mdx:5]:
```suggestion
By default, the session is terminated once the application is in the background for more than `30 seconds`. You can change this default by setting the `autoSessionTrackingInterval` option to a `Duration` of your choosing.
```

File: platform-includes/configuration/auto-session-tracking/flutter.mdx [hunk 5]
```diff
@@ -1,29 +1,23 @@
...
+});
 ```
 
-If you'd like to opt out of this feature, you can do so using options.
+#### Disable Auto Session Tracking
 
-```dart
-import 'package:flutter/widgets.dart';
-import 'package:sentry_flutter/sentry_flutter.dart';
+If you'd like to opt out of capturing sessions, set the option `enableAutoSessionTracking` to `false`. If you disable this feature, release health will not be available.
```
Comment (coolguyzone) [inline platform-includes/configuration/auto-session-tracking/flutter.mdx:16]:
```suggestion
If you'd like to opt out of capturing sessions, set the `enableAutoSessionTracking` option to `false`. If you disable this feature, release health will not be available.
```

File: platform-includes/enriching-events/attach-screenshots/flutter.mdx [hunk 6]
```diff
@@ -1,48 +1,34 @@
...
+  ),
+);
 ```
 
-### Filtering Screenshots
+## Filtering Screenshots
 
-You can filter your screenshots by using the `beforeScreenshot` callback.
-It is called before taking a screenshot and if the callback returns `false`, the screenshot will not be attached.
+You can filter your screenshots by using the `beforeScreenshot` callback which is called before attaching a screenshot to an event. By default, the callback returns `true` which means that all screenshots are attached.
```
Comment (coolguyzone) [inline platform-includes/enriching-events/attach-screenshots/flutter.mdx:20]:
```suggestion
You can filter your screenshots by using the `beforeScreenshot` callback, which is called before attaching a screenshot to an event. By default, the callback returns `true` which means that all screenshots are attached.
```

**Review by coolguyzone — APPROVED** [review]
Added a couple suggestions, looks good! Thanks for updating!

**Pull request merged**